src/views/Signup/utils/signup.test.ts

```diff
@@ -0,0 +1,123 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { UserSettings } from '@internxt/sdk/dist/shared/types/userSettings';
+import signup from './signup';
+import errorService from 'app/core/services/error.service';
+import localStorageService from 'app/core/services/local-storage.service';
+import { planThunks } from 'app/store/slices/plan';
+import { productsThunks } from 'app/store/slices/products';
+import { referralsThunks } from 'app/store/slices/referrals';
+import { userActions, userThunks } from 'app/store/slices/user';
+import envService from 'app/core/services/env.service';
+
+vi.mock(import('app/core/services/error.service'));
+vi.mock(import('app/core/services/local-storage.service'));
+vi.mock(import('app/store/slices/plan'));
+vi.mock(import('app/store/slices/products'));
+vi.mock(import('app/store/slices/referrals'));
+vi.mock(import('app/store/slices/user'));
+vi.mock(import('app/core/services/env.service'));
```

Comment:
We can use `spyOn` instead of mocking some imports here like:
- `error.service`
- `local-storage.service`
- `env.service`